scripts/linters/custom_eslint_checks/rules/protractor-practices.spec.js

```diff
@@ -155,5 +160,12 @@ ruleTester.run('protractor-practices', rule, {
         message: 'Please do not use await for "get()"',
       }],
     },
+    {
+      code:
+      '[].forEach(item)',
```

Comment:
Can you add another test case where the list is a variable (since that's how developers normally call forEach)?